test/fixtures/models/katello_repositories.yml

```diff
@@ -37,6 +37,52 @@ fedora_17_x86_64_dev:
   distribution_bootable: false 
   distribution_uuid: 'xyz123'
 
+fedora_17_x86_64_clone:
+  name:                 Fedora 17 x86_64 clone 
+  pulp_id:              2
+  content_id:           1
+  content_type:         yum
+  label:                fedora_17_x86_64_clone_label
+  relative_path:        '/ACME_Corporation/dev/fedora_17_clone_label'
+  environment_id:       <%= ActiveRecord::Fixtures.identify(:dev) %>
+  product_id:           <%= ActiveRecord::Fixtures.identify(:fedora) %>
+  gpg_key_id:           <%= ActiveRecord::Fixtures.identify(:fedora_gpg_key) %>
+  content_view_version_id: <%= ActiveRecord::Fixtures.identify(:library_default_version) %>
+  url:                 "http://myrepo.com"
+  distribution_arch: "test_arch"
+  distribution_version: "2.1"
+  distribution_family: "Test Family"
+  distribution_variant: "TestVariant"
+  distribution_bootable: true
+  distribution_uuid: 'xyz123'
+
+fedora_17_library_non_default:
+  name:                 Fedora 17 library non default 
+  pulp_id:              fedora_17_library_non_default 
+  content_id:           1
+  content_type:         yum
+  label:                fedora_17_x86_64_library_non_default
+  relative_path:        '/ACME_Corporation/dev/fedora_17_library_non_default_label'
+  environment_id:       <%= ActiveRecord::Fixtures.identify(:library) %>
+  product_id:           <%= ActiveRecord::Fixtures.identify(:fedora) %>
+  gpg_key_id:           <%= ActiveRecord::Fixtures.identify(:fedora_gpg_key) %>
+  content_view_version_id: <%= ActiveRecord::Fixtures.identify(:library_view_version_1) %>
+  url:                 "http://myrepo.com"
+
+fedora_17_dev_non_default:
+  name:                 Fedora 17 dev non default 
+  pulp_id:              fedora_17_dev_non_default 
+  content_id:           1
+  content_type:         yum
+  library_instance:     fedora_17_library_non_default
+  label:                fedora_17_x86_64_dev_non_default
+  relative_path:        '/ACME_Corporation/dev/fedora_17_library_non_default_label'
+  environment_id:       <%= ActiveRecord::Fixtures.identify(:dev) %>
+  product_id:           <%= ActiveRecord::Fixtures.identify(:fedora) %>
+  gpg_key_id:           <%= ActiveRecord::Fixtures.identify(:fedora_gpg_key) %>
+  content_view_version_id: <%= ActiveRecord::Fixtures.identify(:library_view_version_2) %>
+  url:                 "http://myrepo.com"
```

Comment:
when I removed the content view versions that were in staging and dev environments, many of the tests relied on fedora_17_x86_64 being in both environments, which wasn't possible because fedora_17_x86_64 is in the acme default content view version, which couldn't have any versions in dev or staging. It was easier to create two more repos to replicate this behavior.